examples/http_server/module/src/lib.rs

```diff
@@ -29,7 +29,7 @@ oak::entrypoint!(oak_main => |_in_channel| {
     oak::run_event_loop(node, http_channel);
 });
 
-struct StaticHttpServer;
+pub struct StaticHttpServer;
```

Comment:
nit: If you're making this `pub` and re-using it, it would be nice to add a doc comment.